src/ExtendedXmlSerializer/ExtendedXmlSerializerConfig.cs

```diff
@@ -132,6 +134,11 @@ void IExtendedXmlSerializerConfig.WriteObject(XmlWriter writer, object obj)
             _serializer(writer, (T) obj);
         }
 
+        public bool IsSatisfiedBy( Type type )
+        {
+            return specification( type );
```

Comment:
Boo!  I just realized that I did NOT make a default implementation!  I did, but I did not put it in the constructor.  So now I understand your confusion.  I will address this.